client/src/main/java/org/apache/cassandra/sidecar/client/HttpResponseImpl.java

```diff
@@ -32,17 +32,22 @@ public class HttpResponseImpl implements HttpResponse
     private final String statusMessage;
     private final byte[] raw;
     private final Map<String, List<String>> headers;
+    private final SidecarInstance respondingServer;
 
     /**
      * Constructs a response object with the provided values
      *
      * @param statusCode    the status code of the response
      * @param statusMessage the status message of the response
      * @param headers       the headers from the response
+     * @param server        the server that returns the response
```

Comment:
this javadoc seems to reference an inexistent param, and `./gradlew javadocs` will probably fail
```suggestion
     * @param server        the server that returns the response
```